initializr-service/src/main/java/io/spring/initializr/service/extension/SpringCloudStreamRequestPostProcessor.java

```diff
@@ -33,38 +33,53 @@
 
 	private final Dependency springCloudStreamKafka;
 
-	private final Dependency springCloudStreamReactive;
+	private final Dependency springCloudStreamRabbit;
+
+	private final Dependency streamTest;
```

Comment:
I am not against it but I'd like this issue to be focused to the task at hand.

Reply:
Not sure. I thought the task at hand was tidying up the stream dependencies.